service.go

```diff
@@ -0,0 +1,210 @@
+// Package mesg is a service client for mesg-core.
+// For more information please visit https://mesg.com.
+package mesg
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"sync"
+	"time"
+
+	"encoding/json"
+
+	"context"
+
+	"github.com/mesg-foundation/core/api/service"
+	"google.golang.org/grpc"
+)
+
+const (
+	tcpEndpointEnv = "MESG_ENDPOINT_TCP"
```

Comment:
you should use the `MESG_ENDPOINT` env variable. We are going to remove the socket server to keep only the TCP and deprecated `MESG_ENDPOINT_TCP` env.